database/engine/README.md

```diff
@@ -106,4 +106,41 @@ There are explicit memory requirements **per** DB engine **instance**, meaning *
 An important observation is that RAM usage depends on both the `page cache size` and the 
 `dbengine disk space` options. 
 
+## File descriptor requirements
+
+The Database Engine may keep a **significant** amount of files open per instance (e.g. per streaming
+slave or master server). When configuring your system you should make sure there are at least 50
+file descriptors available per `dbengine` instance.
+
+Netdata allocates 25% of the available file descriptors to its Database Engine instances. This means that only 25%
+of the file descriptors that are available to the Netdata service are accessible by dbengine instances.
+You should take that into account when configuring your service
+or system-wide file descriptor limits. You can roughly estimate that the netdata service needs 2048 file
+descriptors for every 10 streaming slave hosts when streaming is configured to use `memory mode = dbengine`.
+
+If for example one wants to allocate 65536 file descriptors to the netdata service on a systemd system
+one needs to edit the netdata systemd file `/etc/systemd/system/multi-user.target.wants/netdata.service`
```

Comment:
As per the netdata-installer.sh, method install_netdata_service() from functions.sh library, we don't put netdata.service under `multi-user.target.wants`


```374                         SYSTEMD_DIRECTORY=""
375 
376                         if [ -d "/lib/systemd/system" ]; then
377                                 SYSTEMD_DIRECTORY="/lib/systemd/system"
378                         elif [ -d "/usr/lib/systemd/system" ]; then
379                                 SYSTEMD_DIRECTORY="/usr/lib/systemd/system"
380                         fi
381 
382                         if [ "${SYSTEMD_DIRECTORY}x" != "x" ]; then
383                                 echo >&2 "Installing systemd service..."
384                                 run cp system/netdata.service "${SYSTEMD_DIRECTORY}/netdata.service" &&
385                                         run systemctl daemon-reload &&
386                                         run systemctl enable netdata &&
387                                         return 0
388                         else
389                                 echo >&2 "no systemd directory; cannot install netdata.service"
390                         fi
```

Reply:
I used `netdata-installer.sh` though and `netdata.service` was put under `multi-user.target.wants`. Please check the host netdata-deb9-test-mfundul.netdata.rocks. There's something we're missing.

Reply:
It's possible that systemctl creates `/etc/systemd/system/multi-user.target.wants/netdata.service` after installing the service.

Reply:
Here it is:
```
lrwxrwxrwx 1 root root 35 Jun 18 09:32 /etc/systemd/system/multi-user.target.wants/netdata.service -> /lib/systemd/system/netdata.service
```
This is probably what `systemctl enable` creates.